File: node.js/백준/Bronze/27160. 할리갈리/할리갈리.js
const [N, ...input] = require('fs')
    .readFileSync(0, 'utf-8')
    .toString()
    .trim()
    .split('\n')

const cards = {
    "STRAWBERRY": 0,
    "BANANA": 0,
    "LIME": 0,
    "PLUM": 0
}

for (let i = 0; i < +N; i++) {
    const [key, value] = input[i].split(' ')
    cards[key] += Number(value)
}

if (cards.BANANA == 5 || cards.LIME == 5 || cards.PLUM == 5 || cards.STRAWBERRY == 5) {
    console.log('YES')
} else {
    console.log('NO')
}

File: Python/백준/Bronze/27160. 할리갈리/할리갈리.py
N = int(input())

cards = {
    "STRAWBERRY": 0,
    "BANANA": 0,
    "LIME": 0,
    "PLUM": 0
}

for i in range(N):
    key,value = input().split()
    cards[key]+= int(value)

if 5 in cards.values():
    print('YES')
else:
    print('NO')